packages/sakuli-core/src/bootstrap/get-node-modules-paths.function.ts

```diff
@@ -0,0 +1,26 @@
+import { join, resolve } from "path";
+import execa from "execa";
+import { isExistingDirectory } from "./is-existing-directory.function";
+
+export const getNodeModulesPaths = async (
+  path: string = "."
+): Promise<string[]> => {
+  let nodeModulesPaths: string[] = [];
+  let previous: string;
+
+  do {
+    const modules = join(path, "node_modules/@sakuli");
+    if (isExistingDirectory(modules)) {
+      nodeModulesPaths.push(modules);
+    }
+
+    previous = path;
+    path = resolve(join(previous, ".."));
+  } while (previous !== path);
+
+  let { stdout } = await execa("npm", ["root", "-g"]);
```

Comment:
I just realize you determine the root path here. I'd recommend to do this earlier in the code and use the following exit condition for directory scanning:

```
while(path !== npmRootPath)
```